app/frontend/components/domains/requirements-library/requirement-field-edit/index.tsx

```diff
@@ -204,7 +204,7 @@ const requirementsComponentMap = {
     return (
       <EditableGroup
         sx={{
-          "& > div:first-child": {
+          "& > div:first-of-type": {
```

Comment:
this fixes a console error.
console.js:273 The pseudo class ":first-child" is potentially unsafe when doing server-side rendering. Try changing it to ":first-of-type".

ChatGPT:
Yes, in most cases, replacing :first-child with :first-of-type is a suitable and straightforward fix to the warning about using :first-child